arduino/smartcar/smartcar.ino

```diff
@@ -149,15 +275,18 @@ void loop()
                          false, 0);
         }
 #endif
-        if (safetyFeatures) // check if the safety system is enabled
-                            //safetyFeatures && frontUltDis <= 150 || safetyFeatures && backIRDis <= 40
+        if (safetyFeatures){// check if the safety system is enabled
+                            //safetyFeatures && frontUltDis <= 150 || safetyFeatures && backUltDis <= 100
                             //Also check if sensors are in range to avoid going through all checks if they aren't
-        {
-            if (!activeAvoidance)
-            {
-                stopZoneAutoBreak(frontUltDis, backIRDis);
+            registerCollision(frontUltDis, leftUltDis, rightUltDis, backUltDis, angleChangeDegree, carSpeed);
+            if (!activeAvoidance){
+                stopZoneAutoBreak(frontUltDis, backUltDis);  
             }
-            incomingAvoidanceThreshold(frontUltDis, backIRDis);
+            incomingAvoidanceThreshold(frontUltDis, backUltDis);
+            
+        } else {
+                driveForwards = true;
+                drivebackwards = true;
         }
 
```

Comment:
I don't know why it's in the master version and not in mine is this a problem

Reply:
#ifdef __SMCE__	#ifdef __SMCE__
@@ -173,95 +302,140 @@ void loop()

Reply:
I made this comment looking at something different so you don't have to worry about this comment when it comes to terms this pull request